test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +35,37 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // empty list
+        assertNotNull();
+
+        // no null objects
+        assertNotNull("ab");
+        assertNotNull("abc","abc");
+        assertNotNull("aa","a","abc");
+
+        // one null object
+        assertIsNull((Object) null);
+
+        // some null objects
+        assertIsNull("abc", null);
+        assertIsNull("abc", null, "aa");
+
+        // all null objects
+        assertIsNull(null, null);
+        assertIsNull(null, null, null);
+
+    }
+
+    private void assertIsNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertNotNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
+
```

Comment:
This may be a matter of personal preference but I would prefer to put the `assertAreUnique` and `assertNotUnique` methods before the `assertIsNull` and `assertNotNull` methods since they are referenced first in the code :P

Reply:
Also, here's something for you to think about. Do you think there can be alternative method names to  `assertIsNull` and `assertNotNull` that may better reflect what they are testing?